### ISSUE TITLE
RTL Audit & Resolution

### ISSUE THREAD

**Issue by ramiy** [WordImpress/Give#736]
See the attached screenshot:

![give add ons page rtl issues](https://cloud.githubusercontent.com/assets/576623/16879155/fe743946-4ab7-11e6-85e1-082cd6f0715c.png)

**Comment by ramiy** [comment]
The plugin needs and RTL audit. Many pages are missing RTL view.

**Comment by ramiy** [comment]
![rtl](https://cloud.githubusercontent.com/assets/576623/16890257/9e572792-4af5-11e6-87ed-d43c78f574c2.png)

**Comment by ramiy** [comment]
Front end issues:

![front1](https://cloud.githubusercontent.com/assets/576623/16893815/3c564940-4b4c-11e6-8a34-b1f0807e2972.png)

![front2](https://cloud.githubusercontent.com/assets/576623/16893818/43f7025c-4b4c-11e6-9b02-51ca5d40496d.png)

**Comment by ramiy** [comment]
![addons-rtl](https://cloud.githubusercontent.com/assets/576623/17086534/6362218c-51fd-11e6-9922-72bed5807bbc.png)

Fixed RTL addons page

**Comment by DevinWalker** [comment]
There are a few lingering issues here after merging in #796 

Still an issue:

![3c564940-4b4c-11e6-8a34-b1f0807e2972](https://cloud.githubusercontent.com/assets/1571635/17112813/7a4dddfa-525c-11e6-833e-21f1535e68a5.png)

**Comment by ramiy** [comment]
@DevinWalker yes I know.

I fixed the admin issues first, it's easier. Now I'm moving to the FrontEnd.

**Comment by DevinWalker** [comment]
Thanks for the update @ramiy it's looking good - I can almost read RTL now 👀

**Comment by ramiy** [comment]
@DevinWalker,

For now the RTL is working. Not 100% but I guess it fixes over 95% of the cases.

Currently, **rtl.scss** is updated manually. In the future we will need to do that automatically using a gulp taks.

**Comment by DevinWalker** [comment]
Hey @ramiy I just reviewed #812 and notice a few items:


1. Should the border and icon be moved to the right for all notices?
![rtl1](https://cloud.githubusercontent.com/assets/1571635/17154902/cd3d7f76-5337-11e6-9d7a-6c98a2290653.png)

2. This border appears to be missing:
![rtl2](https://cloud.githubusercontent.com/assets/1571635/17154916/db65e5f2-5337-11e6-941b-d4cadcbede5d.png)

3. When clicking the donate button the loading icon should not move the button - rather it should appear to the right/left of it:
![rtl3](https://cloud.githubusercontent.com/assets/1571635/17154924/e27ee26c-5337-11e6-9814-25cb28adf786.gif)
![rtl4](https://cloud.githubusercontent.com/assets/1571635/17154923/e26199aa-5337-11e6-9f59-fef76f5736ad.gif)

**Comment by DevinWalker** [comment]
That would be cool to have a gulp task that updated it manually. I added a few items I noticed here:  #736

**Comment by ramiy** [comment]
@DevinWalker Ok, I fixed the warnings and the total-amount border.

**Comment by ramiy** [comment]
As for the gulp task, if I'm not mistaking, It will require us to update the SCSS/CSS depending on the library we will use. Each library has it's own ignore rules and other settings.

**Comment by DevinWalker** [comment]
@ramiy do you have any examples in mind or links to articles about doing this? 👍 

Fixed with merge here: https://github.com/WordImpress/Give/issues/812

**Comment by ramiy** [comment]
https://github.com/jjlharrison/gulp-rtlcss

https://github.com/tepez/gulp-cssjanus
